pkg/model/dev.go

```diff
@@ -614,10 +614,15 @@ func (dev *Dev) expandEnvFiles() error {
 		}
 
 		for name, value := range envMap {
-			dev.Environment = append(
-				dev.Environment,
-				EnvVar{Name: name, Value: value},
-			)
+			if value == "" {
```

Comment:
The code in this function and the one in `translateServiceEnvFile` within translate.go looks very similar. Not in this PR but I think we have to consider to extract the code to a common place to avoid slightly different implementations of the same code as we have/had with `build`.

I'm assuming that the logic is the same. Taking a quick look seems almost the same

Reply:
yes, also with Environment variables